packages/compile-solidity/test/test_oldversions.js

```diff
@@ -21,16 +19,41 @@ const compileOptions = {
   },
   quiet: true
 };
+
 const supplierOptions = {
   solcConfig: compileOptions.compilers.solc,
   events: {
     emit: () => {}
   }
 };
 
+//NOTE: do *not* replace this with an actual .sol
+//file named 'FilenameWith:Colon.sol', as that will
+//prevent the repo from being checked out on Windows!!
+const __OLDCONTRACT = `
+//SPDX-License-Identifier: MIT
+pragma solidity >=0.4.9 <0.4.20;
+
+contract SimpleContract {
+  uint x;
+
+  function SimpleContract() public {
+    x = 7;
+  }
+}
+`;
+
+//I've put a colon in the filename to be extra-sure
+//that this is indeed testing what it's supposed to
+//(that colons in file paths don't screw things up;
+//"project:/" should suffice to trigger the issue but
+//I want to be extra sure)
+const sourcePath = `${workingDirectory}/contracts/FilenameWith:Colon.sol`;
```

Comment:
I wonder if we should use `path.join`, but it shouldn't block this. I mean, that's what the `windows/runner` branch is for.

Reply:
Oh, yeah, probably.  Oh well, like you say, that can be handled later...